### ISSUE TITLE
check-disk-usage.rb fails on Windows with sys-filesystem (1.1.4)

### ISSUE THREAD

**Issue by DracoAter** [sensu-plugins/sensu-plugins-disk-checks#20]
Error:
```
Check failed to run: nil can't be coerced into Float, ["C:/opt/ruby/lib/ruby/gems/2.1.0/gems/sensu-plugins-disk-checks-1.0.3/bin/check-disk-usage.rb:178:in `*'", "C:/opt/ruby/lib/ruby/gems/2.1.0/gems/sensu-plugins-disk-checks-1.0.3/bin/check-disk-usage.rb:178:in `percent_inodes'", "C:/opt/ruby/lib/ruby/gems/2.1.0/gems/sensu-plugins-disk-checks-1.0.3/bin/check-disk-usage.rb:136:in `check_mount'", "C:/opt/ruby/lib/ruby/gems/2.1.0/gems/sensu-plugins-disk-checks-1.0.3/bin/check-disk-usage.rb:120:in `block in fs_mounts'", "C:/opt/ruby/lib/ruby/gems/2.1.0/gems/sensu-plugins-disk-checks-1.0.3/bin/check-disk-usage.rb:111:in `each'", "C:/opt/ruby/lib/ruby/gems/2.1.0/gems/sensu-plugins-disk-checks-1.0.3/bin/check-disk-usage.rb:111:in `fs_mounts'", "C:/opt/ruby/lib/ruby/gems/2.1.0/gems/sensu-plugins-disk-checks-1.0.3/bin/check-disk-usage.rb:196:in `run'", "C:/opt/ruby/lib/ruby/gems/2.1.0/gems/sensu-plugin-1.2.0/lib/sensu-plugin/cli.rb:56:in `block in <class:CLI>'"]
```

The problem is that in sys-filesystem (1.1.4) Sys::Filesystem::Stat object now responds to `:inodes`.
https://github.com/djberg96/sys-filesystem/blob/ffi/lib/windows/sys/filesystem.rb#L103

Can be fixed by chaging the line:
```
if fs_info.respond_to?(:inodes) # needed for windows
```
to
```
if fs_info.respond_to?(:inodes) and !fs_info.inodes.nil? # needed for windows
```

**Comment by eheydrick** [comment]
Hi @DracoAter, thanks for the issue. Can you put together a pull request with the fix and we'll get that updated. Also, you should use `&&` instead of `and`.

**Comment by analytically** [comment]
Fixed in 1.1.1.
